lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -729,9 +729,9 @@ def add_name_only(post, payment_method)
 
       # This surfaces the three_d_secure_usage.supported field and saves it as an instance variable so that we can access it later on in the response
       def add_card_3d_secure_usage_supported(response)
-        return unless response.params['card'] && response.params['card']['three_d_secure_usage']
+        return unless response.params['card'] && response.params['card']['three_d_secure_usage'] && response.params['card']['three_d_secure_usage']['supported']
```

Comment:
We should leave off this final `&&` because we need to be able to return a `false` value

Reply:
ope! Youre totally right thanks for catching this!